src/sidebar.jsx

```diff
@@ -228,7 +228,10 @@ const DropdownWithKebab = ({
         setIsOpen(false);
     };
     const onToggleHidden = () => {
-        setShowHidden(!showHidden);
+        setShowHidden(showHidden => {
+            localStorage.setItem("cockpit-navigator.showHiddenFiles", !showHidden ? "true" : "false");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-214-20240126-171337-3993f500-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)